cmd/frostfs-node/morph.go

```diff
@@ -46,6 +46,7 @@ func initMorphComponents(c *cfg) {
 			c.internalErr <- errors.New("morph connection has been lost")
 		}),
 		client.WithSwitchInterval(morphconfig.SwitchInterval(c.appCfg)),
+		client.WithContext(c.ctx),
```

Comment:
Do we _need_ this context?

Reply:
In current implementation it is redundant, removed.